library/core/src/task/wake.rs

```diff
@@ -172,15 +184,15 @@ pub struct Context<'a> {
 }
 
 impl<'a> Context<'a> {
-    /// Create a new `Context` from a `&Waker`.
+    /// Create a new [`Context`] from a [`&Waker`](Waker).
```

Comment:
As far as I’m aware, I believe there’s commonly *no* links to the type itself in an inherent method. I.e. the `Context` being a link isn’t really useful at all.
```suggestion
    /// Create a new `Context` from a [`&Waker`](Waker).
```

I’m also adding comments to other places where a link links to the place we’re already at.

Reply:
All gone. Sorry — I actually noticed they were being excluded but not soon enough to not have made some of those changes.

Personally, I'd rather live in a world where all references to a symbol are machine-readable (whether or not they actually _become links_) but I recognize there isn't much value in trying to achieve that given current tooling (e.g. rust-analyzer does not rename in docs).